loadtestconfig.rst

```diff
@@ -0,0 +1,571 @@
+
+The variables are as follow below. 
+
+    
+
+
+ServerURL 
+-------------------------------------------------------
+
+    ``ServerURL`` is the URL for the the load test.
+    It should be the public facing URL of the Mattermost instance.
+    
+    -example    ``"ServerURL":http//localhost:8065``
+    
+
+WebsocketURL  
+-----------------------------------------------------
+
+    ``WebsocketURL`` is in most cases the same URL as ServerURL with `http`
+    replaced with `ws` or `https` replaced with `wss`.
+
+    -example    ``"WebsocketURL":ws//localhost:8065``
+
+PProfURL 
+-----------------------
+
+    ``PProfURL`` is the URL for the profiling server.
+
+    -example    ``"PProfURL":http//localhost:8067/debug/pprof``
+    
+    
+    .. tip:: Package pprof in the *Go* API writes runtime profiling data in the format expected by the pprof visualization tool. For further inquiry visit the `Go API <https://golang.org/pkg/net/http/pprof/>`_.
+    
+    
+
+DBEndpoint
+------------------------
+
+    ``DBEndpoint`` is the database endpoint for the load test configuration.
+    
+
+    -example    ``"DBEndpoint":mmuserIjqOyuhpSmBQwFOrl@tcp(loadtest1ar.cwr8afqyinx1.us-east-1.rds.amazonaws.com:3306)/mattermost?charset=utf8mb4utf8&readTimeout=20s&writeTimeout=20s&timeout=20s``
+
+LocalCommands 
+--------------------------
+
+    ``LocalCommands`` runs Mattermost CLI commands locally instead of going through SSH. Set this configuration to true if you are running the load tests on the same machine as one of the application servers.
+    
+    -options    true/false
+    -default    true
+
+
+SSHHostnamePort 
+----------------------------
+
+    ``SSHHostnamePort`` is the hostname and port of any one of the application servers you are testing. Mattermost CLI commands are run here for the test.
+    
+    -example    ``"SSHHostnamePort":2222``
+
+SSHUsername
+-----------------------
+
+    ``SSHUsername`` is the username for the SSH authentication.
+    
+    -example    ``"SSHUsername":spellChristopher``
+
+SSHPassword
+------------------------
+
+    ``SSHPassword`` is the SSH password to connect with or "" if using a key.
+    
+    -example    ``"SSHPassword":yellowsubmarine07``
+
+    
+
+SSHKey
+-------------
+
+    ``SSHKey`` is the file path of the SSH key used to connect.
+    
+    -example    ``"SSHKey":12345abcd``
+    
+    .. where would and ssh key typically go in a Mattermost install?
+
+MattermostInstallDir 
+-------------------------------
+
+    ``MattermostInstallDir`` is the location of the Mattermost installation
+    directory. It must be the directory on the machine from which the CLI commands run.
+    This is determined by the LocalCommands_ setting.
+
+
+    -example    ``"MattermostInstallDir":/home/christopher/go/src/github.com/mattermost/mattermost-server/dist/mattermost``
```

Comment:
Let's change the samples here to not include my name. Let's use `/opt/mattermost` instead.